kubernetes/config/common.yaml

```diff
@@ -31,17 +39,42 @@ data:
                   backend: { grpc: { address: 'storage-1.storage.buildbarn:8981' } },
                   weight: 1,
                 },
+                {
+                  backend: { grpc: { address: 'storage-2.storage.buildbarn:8981' } },
+                  weight: 1,
+                },
+                {
+                  backend: { grpc: { address: 'storage-3.storage.buildbarn:8981' } },
+                  weight: 1,
+                },
               ],
             },
           },
         },
       },
+      blobstoreFrontend: {
+        contentAddressableStorage: {
+          grpc: { address: 'frontend.buildbarn:8980' },
+        },
+        actionCache: {
+          grpc: { address: 'frontend.buildbarn:8980' },
+        },
+      },
       # Remember to set your browserUrl to the ingress of the browser deployment
-      # browserUrl: 'http://address-of-bb-browser-ingress:80',
+      browserUrl: 'https://bb-browser.example.com',
       maximumMessageSizeBytes: 16 * 1024 * 1024,
-      httpListenAddress: ':80'
+      httpListenAddress: ':80',
```

Comment:
Is this option still used anywhere? I mean, all binaries take `global` now, right?